lib/validation/schema/specVersion/kind/project.json

```diff
@@ -253,6 +253,77 @@
 				}
 			}
 		},
+		"builder-bundles-3.0": {
+			"type": "array",
+			"additionalProperties": false,
+			"items": {
+				"type": "object",
+				"additionalProperties": false,
+				"properties": {
+					"bundleDefinition": {
```

Comment:
There are no changes to the `bundleDefinition`, so this section could be defined separately and then re-used instead of copied.